protocol/requests/create_account.go

```diff
@@ -74,6 +76,10 @@ func ValidateAccountCreationRequest(c CreateAccount) error {
 		return ErrCreateAccountInvalidDisplayName
```

Comment:
This can be removed, I suppose?

Reply:
I meant the `if len(c.DisplayName) == 0` but wasn't able to reach this line in code review.

Reply:
Ah, I see. Then maybe we could parameterize `ValidateDisplayName`. Not a big deal though.